_sass/minima.scss

```diff
@@ -1,3 +1,5 @@
+// This file is unmodified but must be included so our custom syntax-highlighting partial is read
```

Comment:
This says the file is unmodified but there are actually two small changes.  Was that intentional?